managedtenants/bundles/index_builder.py

```diff
@@ -88,3 +94,75 @@ def _push(self, index_image):
             err_msg = f"failed to push {index_image}: {e}."
             self.log.error(err_msg)
             raise IndexBuilderError(err_msg)
+
+    def _validate_sql_catalog(self, tag):
+        """
+        Validates an sql based index_image. Makes sure it contains bundles.
+        """
+        sql_catalog = None
+        try:
+            in_memory_db = self.docker_api.extract_file_from_container(
+                tag, "/database/index.db"
+            )
```

Comment:
Can this be it's own `try` block for clarity?

Reply:
fixed